.github/workflows/ci_workflows.yml

```
@@ -50,13 +50,13 @@ jobs:
       pytest: false
 
       envs: |
-        - windows: py39-notebooks
-        - macos: py38-notebooks
-        - linux: py310-notebooks
+        - windows: py310-notebooks
+        - macos: py311-notebooks
+        - linux: py313-notebooks
 
         - linux: py311-docs
-        - macos: py39-docs
-        - windows: py38-docs
+        - macos: py310-docs
+        - windows: py10-docs
```

```suggestion
        - windows: py313-docs
```
I assume this was meant to be `py310`, but could as well try a newer version here.